templates/quickstart-cloudwatch-dashboard.yaml

```diff
@@ -0,0 +1,42 @@
+AWSTemplateFormatVersion: 2010-09-09
```

Comment:
Is the dev flow:
* Update the config json
* Run the makefile
* Commit the config json and the generated template

?

Reply:
If we go this way forward (see my comment above), this needs to go to `README` and ideally some kind of automated pre-commit check.

Reply:
I've just pushed a `development.md` file that details the process. Let me know how this looks!